src/views/shared/snackbar/snackbar.view.tsx

```diff
@@ -51,14 +52,16 @@ export const Snackbar: FC<SnackbarProps> = ({ message, onClose, onReport }) => {
         <div className={classes.wrapper}>
           <Icon message={message} />
           <p className={classes.message}>{text}</p>
```

Comment:
We should not ask `Would you mind reporting it?` when the report feature is not enabled.

<img width="687" alt="image" src="https://user-images.githubusercontent.com/15896177/227305746-8cf52c93-7f9b-4ad8-8369-933efc978caf.png">

Reply:
Done!